### PR TITLE
blockdev_inc:add nbd support for inc backup

### PR DESCRIPTION
**Pull request by aliang123**
Add backend nbd support for incremental live  backup

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:2070415

### CODE REVIEW

**Comment by aliang123** [comment]
(01/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_no_bitmap.q35: PASS (61.90 s)
(02/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_cluster.inc_cluster_size.size_512.q35: PASS (99.79 s)
(03/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_cluster.inc_cluster_size.size_4K.q35: PASS (105.77 s)
(04/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_cluster.inc_cluster_size.size_2M.q35: PASS (106.65 s)
(05/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_cluster.inc_cluster_size.q35: PASS (105.40 s)
(06/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_remove_bitmap.q35: PASS (64.52 s)
(07/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_without_bitmapmode.q35: PASS (60.79 s)
(08/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup_nonexist_target.q35: PASS (50.45 s)
(09/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.simple_test.with_data_plane.q35: PASS (94.03 s)
(10/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.simple_test.q35: PASS (98.96 s)
(11/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.simple_test.src_cluster_size_512.with_data_plane.q35: PASS (155.79 s)
(12/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.simple_test.src_cluster_size_512.q35: PASS (98.70 s)
(13/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.simple_test.auto_compress.dst_cluster_size_512.with_data_plane.q35: PASS (255.17 s)
(14/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.simple_test.auto_compress.dst_cluster_size_512.q35: PASS (315.29 s)
(15/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.simple_test.auto_compress.dst_random_cluster_size.with_data_plane.q35: PASS (106.52 s)
(16/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.simple_test.auto_compress.dst_random_cluster_size.q35: PASS (127.73 s)
(17/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.simple_test.manual_completed.auto_compress.dst_cluster_size_2M.with_data_plane.q35: PASS (96.22 s)
(18/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.simple_test.manual_completed.auto_compress.dst_cluster_size_2M.q35: PASS (98.67 s)
(19/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.simple_test.manual_completed.auto_compress.src_cluster_size_2M.with_data_plane.q35: PASS (103.56 s)
(20/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.simple_test.manual_completed.auto_compress.src_cluster_size_2M.q35: PASS (108.11 s)
(21/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.during_reboot.with_data_plane.q35: PASS (110.75 s)
(22/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.during_reboot.q35: PASS (117.53 s)
(23/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.during_stress.with_data_plane.q35: PASS (105.43 s)
(24/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup.during_stress.q35: PASS (107.94 s)
(25/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_clear_bitmap.q35: PASS (62.42 s)
(26/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_nospace_with_bitmap_mode.on_success.q35: PASS (90.81 s)
(27/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_nospace_with_bitmap_mode.always.q35: PASS (87.83 s)
(28/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_bitmap_mode.bitmap_always.q35: PASS (86.26 s)
(29/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_bitmap_mode.bitmap_never.q35: PASS (89.07 s)
(30/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_bitmap_mode.bitmap_success.q35: PASS (88.15 s)
(31/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_bitmap_mode.bitmap_top.q35: PASS (105.96 s)
(32/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_add_disabled_bitmap.q35: PASS (64.42 s)
(33/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_bitmap_with_granularity.granularity_512.q35: PASS (56.90 s)
(34/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_bitmap_with_granularity.granularity_2G.q35: PASS (57.21 s)
(35/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_bitmap_with_granularity.granularity_between_512_2G.q35: PASS (57.51 s)
(36/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup_invalid_max_workers.q35: PASS (46.75 s)
(37/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_without_bitmap.q35: PASS (33.85 s)
(38/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_add_bitmap_to_raw.q35: PASS (34.76 s)
(39/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_no_space.q35: PASS (204.78 s)
(40/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_expose_active_bitmap.q35: PASS (50.56 s)
(41/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_disable_bitmap.q35: PASS (67.34 s)
(42/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_merge_to_nonexist_bitmap.q35: PASS (52.35 s)
(43/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_merge_external_bitmaps.q35: PASS (125.43 s)
(44/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_merge_bitmaps_with_diff_granularity.q35: PASS (53.06 s)
(45/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_bitmap_with_hotplug.q35: PASS (63.98 s)
(46/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_sync_bitmap_nobitmap.q35: PASS (64.34 s)
(47/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_bitmap_inuse.q35: PASS (69.42 s)
(48/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.differential_backup.q35: PASS (71.83 s)
(49/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_after_commit.q35: PASS (107.02 s)
(50/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_merge_with_nonexist_bitmap.q35: PASS (53.30 s)
(51/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_bitmap_vm_crash_reboot.q35: PASS (94.94 s)
(52/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup_x_perf.q35: PASS (100.62 s)
(53/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_never.q35: PASS (64.14 s)
(54/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_non_persistent_bitmap.q35: PASS (65.38 s)
(55/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup_invalid_sync_mode.q35: PASS (50.77 s)
(56/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_filternode.q35: PASS (97.63 s)
(57/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_enable_bitmap.q35: PASS (65.45 s)
(58/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_always.q35: PASS (64.62 s)
(59/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup_multi_disks.with_data_plane.q35: PASS (119.32 s)
(60/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup_multi_disks.q35: PASS (120.85 s)
(61/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_with_throttling.q35: PASS (159.88 s)
(62/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_modify_backing_bitmaps.q35: PASS (36.24 s)
(63/56) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_stream_dirty_bitmap.q35: PASS (64.69 s)
(64/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_with_ignore.q35: ERROR: QMP command 'blockdev-add' failed    (arguments: {'node-name': 'file_inc', 'driver': 'file', 'filename': '/var/tmp/mnt/inc.qcow2', 'aio': 'threads', 'auto-read-only': True, 'discard': 'unmap'},    error message: {'class': 'GenericError', 'desc': "Could no... (59.92 s)
(65/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_with_guest_agent.q35: PASS (102.30 s)
(66/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.live_backup_add_bitmap.with_raw.persistent_off.q35: PASS (74.18 s)
(67/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.live_backup_add_bitmap.with_raw.persistent_default.q35: PASS (120.15 s)
(68/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup_with_bitmap.q35: PASS (43.80 s)
(69/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup.granularity_512.q35: PASS (98.65 s)
(70/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup.granularity_8K.q35: PASS (102.42 s)
(71/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup.granularity_16M.q35: PASS (158.97 s)
(72/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup.granularity_64M.q35: PASS (104.72 s)
(73/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup.granularity_2G.q35: PASS (170.93 s)
(74/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup.granularity_random.q35: PASS (581.65 s)
(75/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup.q35: PASS (103.67 s)
(76/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup.multi_data_disks.q35: PASS (128.33 s)
(77/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup.multi_data_disks.grouped_complete.q35: PASS (131.68 s)
(78/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup.multi_data_disks.negative_test.q35: PASS (84.29 s)
(79/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup.multi_data_disks.manual_complete.q35: PASS (123.51 s)
(80/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup.multi_data_disks.manual_complete.grouped_complete.q35: PASS (129.64 s)
(81/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_target_not_exist.q35: PASS (31.10 s)
(82/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_with_migration.q35: PASS (131.71 s)
(83/83) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_inc_success.q35: PASS (104.56 s)

All pass except case:blockdev_inc_backup_with_ignore, will update the case after fix the error issue.

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_with_ignore.q35: PASS (143.40 s)

**Comment by aliang123** [comment]
@zhencliu @MiriamDeng or @ngu-niny, please help to review, thanks.

**Comment by MiriamDeng** [comment]
git am 3270.patch
Applying: blockdev_inc_backup:add nbd support for inc backup
The test is on going and once it finishes, will update it.

**Comment by MiriamDeng** [comment]
> git am 3270.patch Applying: blockdev_inc_backup:add nbd support for inc backup The test is on going and once it finishes, will update it.

Update the results to the bug and got one test fail. @aliang123 , could you help to check it ?

**Comment by MiriamDeng** [comment]
LGTM

File: qemu/tests/cfg/blockdev_full_backup_nonexist_target.cfg [hunk 1]
```diff
@@ -29,3 +28,11 @@
...
         enable_ceph_dst1 = no
         image_format_src1 = raw
+    nbd:
+        image_format_src1 = raw
+        image_size_src1 = 2G
+        nbd_port_src1 = 10831
+        force_create_image_src1 = no
+        remove_image_src1 = no
+        enable_nbd_dst1 = no
+    image_size_dst1 = ${image_size_data1}
```
Comment (zhencliu) [inline qemu/tests/cfg/blockdev_full_backup_nonexist_target.cfg:38]: image_size_data1 -》 image_size_src1

**Comment by aliang123** [comment]
(01/11) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_merge_to_nonexist_bitmap.q35: PASS (49.81 s)
(02/11) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup_invalid_max_workers.q35: PASS (49.30 s)
(03/11) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_add_disabled_bitmap.q35: PASS (63.02 s)
(04/11) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_with_ignore.q35: ERROR: QMP command 'blockdev-add' failed    (arguments: {'node-name': 'file_inc', 'driver': 'file', 'filename': '/var/tmp/mnt/inc.qcow2', 'aio': 'threads', 'auto-read-only': True, 'discard': 'unmap'},    error message: {'class': 'GenericError', 'desc': "Could no... (62.04 s)
(05/11) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup_invalid_sync_mode.q35: PASS (46.11 s)
(06/11) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_non_persistent_bitmap.q35: PASS (59.16 s)
(07/11) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup_x_perf.q35: PASS (104.71 s)
(08/11) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_merge_with_nonexist_bitmap.q35: PASS (51.56 s)
(09/11) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup_nonexist_target.q35: PASS (46.82 s)
(10/11) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_add_bitmap_to_raw.q35: PASS (31.81 s)
(11/11) Host_RHEL.m9.u1.nbd.raw.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.x86_64.io-github-autotest-qemu.blockdev_full_backup_with_bitmap.q35: PASS (49.56 s)

blockdev_inc_backup_with_ignore sometimes failed for a auto bug.

**Comment by zhencliu** [comment]
LGTM

**Pull request merged**